frontend/src/metabase/visualizations/lib/action.js

```diff
@@ -15,7 +16,13 @@ export function performAction(action, { dispatch, onChangeCardAndRun }) {
     const url = action.url();
     const ignoreSiteUrl = action.ignoreSiteUrl;
     if (url) {
-      open(url, { ignoreSiteUrl });
+      open(url, {
+        openInSameOrigin: (url, location) => {
+          dispatch(push(location));
+          dispatch(setParameterValuesFromQueryParams(location.query));
```

Comment:
There seems to be no clean way to implement the fix, but this approach seems to be the most straightforward.

Reply:
I'm not entirely sure if this line has anything to do with the fix. Could you elaborate on this? I tried removing this line and the bug still seems fixed.

I think it has something to do with dashboard parameters, but I'm not totally sure.

Reply:
Yes, that's it